packages/shared/src/components/cards/PostMetadata.tsx

```diff
@@ -1,30 +1,29 @@
 import React, { ReactElement, ReactNode, useMemo } from 'react';
 import classNames from 'classnames';
-import { Post } from '../../graphql/posts';
+import { Post, ReadHistoryPost } from '../../graphql/posts';
 import { postDateFormat } from '../../lib/dateFormat';
-import classed from '../../lib/classed';
-
-const Separator = classed(
-  'div',
-  'mx-1 w-0.5 h-0.5 rounded-full bg-theme-label-tertiary',
-);
+import { Separator } from './common';
 
 interface PostMetadataProps {
-  post: Post;
+  createdAt?: string;
+  readTime?: number;
+  numUpvotes?: number;
   className?: string;
   children?: ReactNode;
   typoClassName?: string;
 }
 
 export default function PostMetadata({
-  post,
+  createdAt,
+  readTime,
+  numUpvotes,
```

Comment:
The numUpvotes is now never passed?

Reply:
Yes, ReadingHistoryItem is passing numUpvotes.